### PR TITLE
Hotfix: merchant_name fallback

### PR DESCRIPTION
**Pull request by ivyastrix**
Use merchant_name for expenses and revenues if we have it, otherwise just use name.

### CODE REVIEW

File: pages/api/expense-sources.js [hunk 1]
```diff
@@ -18,7 +18,11 @@ async function getExpenseSources(req, res) {
 
     transactions.forEach((transaction) => {
         if (transaction.amount > 0) {
-            const name = transaction.name;
+            let name = "";
+            if(transaction.merchant_name)
```
Comment (hone1er) [inline pages/api/expense-sources.js:22]: Missing {} on if else statements

File: pages/api/revenue-sources.js [hunk 2]
```diff
@@ -19,7 +19,13 @@ async function getRevenueSources(req, res) {
     transactions.forEach((transaction) => {
 
             if (transaction.amount < 0) {
-                const name = transaction.name;
+
+                let name = "";
+                if(transaction.merchant_name)
+                    name = transaction.merchant_name;
```
Comment (hone1er) [inline pages/api/revenue-sources.js:25]: Same for these if/else missing {}

**Review by hone1er — CHANGES_REQUESTED** [review]
Couple of if/else statements need {} added

File: pages/api/expense-sources.js (hunk 1 above)
Comment (ivyastrix) [reply pages/api/expense-sources.js:22]: doesn't need brackets if there's only one line of code to execute!

File: pages/api/revenue-sources.js (hunk 2 above)
Comment (ivyastrix) [reply pages/api/revenue-sources.js:25]: doesn't need brackets if there's only one line of code to execute!

File: pages/api/expense-sources.js (hunk 1 above)
Comment (hone1er) [reply pages/api/expense-sources.js:22]: 🤯

File: pages/api/revenue-sources.js (hunk 2 above)
Comment (hone1er) [reply pages/api/revenue-sources.js:25]: Oh my code is going to be so much prettier from now on lol

**Pull request merged**